src/components /FormMood.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import {
+  BsEmojiSmile,
+  BsEmojiExpressionless,
+  BsEmojiFrown,
+} from "react-icons/bs";
+
+const FormMood = (props) => {
```

Comment:
example of using ...props to get all remaining props
```suggestion
const FormMood = ({ children, value, text, ...props }) => {
```